server/paths/apply.ts

```diff
@@ -10,6 +10,7 @@ const paths = {
     new: applicationsPath.path('new'),
     start: applicationsPath.path('start'),
     people: applicationsPath.path('people').path('find'),
+    personRisks: applicationsPath.path('people').path(':crn').path('risks'),
```

Comment:
Is this path used anywhere?

Reply:
Good eyes! It was used in the test but not in the code.